content/applications/inventory_and_mrp/inventory/shipping_receiving/reservation_methods/at_confirmation.rst

```diff
@@ -0,0 +1,86 @@
+===========================
+At confirmation reservation
+===========================
+
+.. _inventory/reservation_methods/at-confirmation:
+
+.. |SO| replace:: :abbr:`SO (Sales Order)`
+
+The *At Confirmation* reservation method reserves products **only** when a sales order (SO) is
+confirmed, **and** if enough stock of the products included in the |SO| is already available.
+
+.. seealso::
+   :doc:`About reservation methods <../reservation_methods>`
+
```

Comment:
Suggestion to move this seealso to the bottom of the doc :)
```suggestion
.. seealso::
   :doc:`About reservation methods <../reservation_methods>`

```

Reply:
Suggestion implemented ✅